drivers/display/Kconfig.st7701

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2025 Arduino SA
+# SPDX-License-Identifier: Apache-2.0
+
+config ST7701
+	bool "ST7701 display driver"
```

Comment:
bool "ST7701 display controller driver". We do not write drivers for displays.

Reply:
Apparently you do: https://github.com/zephyrproject-rtos/zephyr/blob/bec4a8474934ef6ad4fd4154314cd8234786a279/drivers/display/Kconfig.st7796s#L5

Quick search shows 60+ results, but okay.